tests/pipelines/eks/awscli-eks-cl2-load.yaml

```diff
@@ -26,6 +30,15 @@ spec:
     taskRef:
       kind: Task
       name:  slack-notification
+  - name: awscli-vpc-create
+    params:
+    - name: stack-name
+      value: $(params.vpc-stack-name)
+    - name: vpc-cfn-url
+      value: "$(params.vpc-cfn-url)"
```

Comment:
Had a discussion offline regarding cleaning up VPC after each run. 
Harish will take a call if we should clean up VPC as well.